index.ios.js

```diff
@@ -1,4 +1,4 @@
 import { AppRegistry } from 'react-native';
-import { App } from './main';
+import { App } from './build';
```

Comment:
now App will be inside of the build folder, I would expect `./build/app`, How does this work? is it loading the index file?